scout/server/blueprints/institutes/templates/overview/institute_settings.html

```diff
@@ -43,11 +43,6 @@
   <script src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/js/bootstrap-select.min.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/select2@4.0.12/dist/js/select2.min.js"></script>
   <script>
-    $(document).ready(function(){
```

Comment:
This was giving error since there is no hide_div function in this page

Reply:
Right, that part I saw and fixed in the evening in the other PR, but I didn't have the energy to think about the tuple sort then. Well done! 👍